### DISCUSSION TITLE
Duplicates identify for mailpiller use space or resources?

### DISCUSSION THREAD

**Discussion by ediazrod** [jsuto/piler#183] [General]
Hi, I lost my mailpiler system on a crash of hard disk (backup are important) :-P

Any way is not critical because I have a backup of the mail system years back, I have all the information and the mails but...

I am importing the mails form the users but, I see a lot of duplicates (may mails has the same id because are the same), because I import the mails by years, is this a issue?.. I don't mind have duplicates if don't get space or resources from my system(I repeat they are identify as duplicates for mailpiler and this is normal I don't care has a lot of duplicartes as far don't waist space or resources..)..

If this is a ussue I can test with mailimport if there is a duplicate by header-id and don't try to import it, but as far as keep the code of import simple I like to ask this question...

**Comment by jsuto** [comment]
Assuming you are importing all your mailboxes and emails, it's quite normal to see some duplicates. The reason is usually that a single email has more recipients. I'd say don't worry much, however after the import compare the archive's size and the # of stored emails before the crash/data loss.
